README.md

```diff
@@ -110,10 +121,10 @@ All [puppet metaparameters](https://docs.puppetlabs.com/references/latest/metapa
 
 ## Limitations
 
-- DSC Composite Resources are not yet fully supported.
+- DSC Composite Resources are not supported.
 - PSObjects like 'PSCredential' as parameters value not yet supported.
```

Comment:
I prefer not supported. We don't know if we will, so it's best not to even state yet.